### ISSUE TITLE
Won't install

### ISSUE THREAD

**Issue by grahamengland** [missedone/yatb#1]
When I try to install this on my Eclipse Neon, I get the following error...
An error occurred while collecting items to be installed
session context was:(profile=C__Users_C054737_eclipse_java-neon_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Problems downloading artifact: org.eclipse.update.feature,com.github.eclipse.yatb.features,1.0.0.201603281533.
MD5 hash is not as expected. Expected: 5942277985994f3fa32d7a88e8adcee8 and found 92bdaf90137c4ed65f4a7ba5ca353517.

**Comment by missedone** [comment]
hi @grahamengland , sounds like network issue.
could you try again?
if still having issue, you can try offline installation: 
git clone this git repo, then in your eclipse -> Help -> Install New Software -> Add -> Local -> locate the folder `dist` of this project

**Comment by grahamengland** [comment]
Thanks Nick.  I downloaded the files locally and it looked like it installed without any errors, but even after restarting Eclipse I don't see the button anywhere... am I missing something?  Not a biggie though, I can just terminate java/javaw processes through the Task Manager.

**Comment by missedone** [comment]
You can see additional terminate button on console view along with the existing red square terminate button.

I will attach a screenshot later, not on laptop right now.

**Comment by grahamengland** [comment]
Thanks - I see it now (red square with grey cross below and to right of it).  Oddly enough it doesn't seem to stop the called java.exe task, but I'm not entirely surprised, I'm doing some pretty odd things in my application.

**Comment by missedone** [comment]
Right, for windows I just delegate the terminate action to the default eclipse implementation, no force terminate like for Linux/Mac
